package/boot/uboot-rockchip/Makefile

```diff
@@ -79,6 +93,15 @@ UBOOT_CONFIGURE_VARS += USE_PRIVATE_LIBGCC=yes
 UBOOT_MAKE_FLAGS += \
   BL31=$(STAGING_DIR_IMAGE)/$(ATF)
 
+define Build/Prepare
+	$(call Build/Prepare/Default)
+
+ifneq ($(DDR_1G),)
+	sed -i "16s/"rk3399-sdram-lpddr4-100.dtsi"/"rk3399-sdram-ddr3-1866.dtsi"/" $(PKG_BUILD_DIR)/arch/arm/dts/rk3399-nanopi-r4s-u-boot.dtsi
```

Comment:
What if someone build normal version AND 1G version together?

Reply:
This only build the firmware for the 1gb ram, firmware names are different like this:
```
openwrt-rockchip-armv8-friendlyarm_nanopi-r4s-1g-ext4-sysupgrade.img.gz
openwrt-rockchip-armv8-friendlyarm_nanopi-r4s-1g-squashfs-sysupgrade.img.gz
openwrt-rockchip-armv8-friendlyarm_nanopi-r4s-ext4-sysupgrade.img.gz
openwrt-rockchip-armv8-friendlyarm_nanopi-r4s-squashfs-sysupgrade.img.gz
```